scripts/reset-simulators.sh

```diff
@@ -1,59 +1,43 @@
 #!/bin/bash
 
-set -o pipefail
```

Comment:
Just stumbling on through when something fails in an unexpected way seems undesirable.